lab_1/main.py

```diff
@@ -25,7 +34,20 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    if type(tokens) == dict:
```

Comment:
why such specific checks? better write general one `if not isinstance(tokens, list): return []` and avoid so many checks below